### PR TITLE
Create Antares-Cygnus.netkan

### PR DESCRIPTION
**Pull request by Felger**

### CODE REVIEW

File: NetKAN/Antares-Cygnus.netkan [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "identifier"    : "Antares-Cygnus",
+    "name"          : "Antares-Cygnus",
+    "author"        : ["Raidernick","Kartoffelkuchen"],
+    "abstract"      : "A joint release between Kartoffelkuchen and Raidernick combining our Cygnus and Antares mods.",
+    "resources": {
+        "homepage"  : "https://github.com/KSP-RO/Antares-Cygnus",
+    },
+    "$kref"         : "#/ckan/github/KSP-RO/Antares-Cygnus",
+    "license"       : "CC-BY-NC-ND-3.0",
+    "ksp_version"   : "1.0"
```
Comment (techman83) [inline NetKAN/Antares-Cygnus.netkan]: @Felger - Missing a comma here

File: NetKAN/Antares-Cygnus.netkan [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "install"       : [
+    {
+        "find"      : "KK_Antares",
+        "install_to": "GameData",
```
Comment (techman83) [inline NetKAN/Antares-Cygnus.netkan]: @Felger - Extraneous comma here

File: NetKAN/Antares-Cygnus.netkan [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+        "find"      : "RN_Cygnus",
+        "install_to": "GameData",
```
Comment (techman83) [inline NetKAN/Antares-Cygnus.netkan]: @Felger - And here :-)

**Comment by plague006** [comment]
`find` requires spec_ver `v1.4`.

To give previous users of `AntaresCygnus` an upgrade path, can you change the identifier to match and also give an epoch bump `"x_netkan_epoch": "1",`.

**Comment by plague006** [comment]
The .zip  has ships in it, we can install those `find: "VAB", install_to: "Ships"`

**Comment by Felger** [comment]
Understood, the mod author isn't maintaining the ships anymore, so he'd rather they weren't installed by default.  This goes for the rest of the Raidernick mods in the recent set as well.

**Comment by plague006** [comment]
Passws local and Jenkins. Merging with my thanks.

**Pull request merged**